app/controllers/aspects_controller.rb

```diff
@@ -11,18 +11,13 @@ class AspectsController < ApplicationController
 
   def create
     @aspect = current_user.aspects.build(aspect_params)
-    aspecting_person_id = params[:aspect][:person_id]
+    aspecting_person_id = params[:person_id]
 
     if @aspect.save
-      flash[:notice] = I18n.t('aspects.create.success', :name => @aspect.name)
-
-      if current_user.getting_started || request.referer.include?("contacts")
-        redirect_to :back
-      elsif aspecting_person_id.present?
+      if aspecting_person_id.present?
         connect_person_to_aspect(aspecting_person_id)
-      else
-        redirect_to contacts_path(:a_id => @aspect.id)
       end
+      render :json => { :id => @aspect.id, :name => @aspect.name }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { detected.<br>Space inside } detected.